packages/node_modules/@ciscospark/plugin-messages/src/messages.js

```diff
@@ -95,14 +107,34 @@ const Messages = SparkPlugin.extend({
    * @param {Object} activity from mercury
    * @returns {Object} constructed event
    */
-  createMessagesEventData(activity) {
+  messageCreatedEventData(activity) {
     return {
       id: constructHydraId(hydraTypes.MESSAGE, activity.id),
       roomId: constructHydraId(hydraTypes.ROOM, activity.target.id),
       roomType: activity.target.tags.includes('ONE_ON_ONE') ? 'direct' : 'group',
       text: activity.object.displayName,
       personId: constructHydraId(hydraTypes.PEOPLE, activity.actor.id),
       personEmail: activity.actor.emailAddress || activity.actor.entryEmail,
+      personOrgId: constructHydraId(hydraTypes.ORGANIZATION, activity.actor.orgId),
```

Comment:
This isn't part of the [Hydra payload](https://developer.webex.com/docs/api/v1/messages/get-message-details) from what I can tell. We have to be careful about including things out-of-spec. Maybe there is a property object we can group things into like `experimental: {}`.

Reply:
You are right.   OrgId is there on the membership, but NOT the messages.   I'll remove this.